src/gui/ExportProjectDialog.cpp

```diff
@@ -142,11 +142,11 @@ OutputSettings::StereoMode mapToStereoMode(int index)
 	switch (index)
 	{
 	case 0:
-		return OutputSettings::StereoMode_Stereo;
+		return OutputSettings::StereoMode_Mono;
 	case 1:
-		return OutputSettings::StereoMode_JointStereo;
+ 		return OutputSettings::StereoMode_Stereo;
```

Comment:
Please remove a space at the beginning of this line.
```suggestion
		return OutputSettings::StereoMode_Stereo;
```